packages/backend/src/managers/modelsManager.ts

```diff
@@ -173,6 +175,7 @@ export class ModelsManager {
       });
 
       try {
+        this.telemetry.logUsage('model.download', { 'model.id': model.id });
```

Comment:
Would be good to have a single event: here we will have 2 events in case of error